examples/platforms/da15000/uart.c

```diff
@@ -100,13 +100,12 @@ void UartBuffClear(void)
 
     volatile uint8_t aBuf;
 
-    while (1)
+    while (hw_uart_read_buf_empty(HW_UART1) != 1)
```

Comment:
This should be a test for false rather then a test for not 1.  While true is usually equal to 1, it doesn't have to be.  C defines true as being not false, and false as 0, so true could be any non-zero value.
I would suggest simply using a not:
```C
while ( ! hw_uart_read_buf_empty(HW_UART1) )
```
or, alternatively
```C
while ( hw_uart_read_buf_empty(HW_UART1) == 0 )
```